### ISSUE TITLE
oss: collect standard library strings (i.e., msvcrt)

### ISSUE THREAD

**Issue by williballenthin** [mandiant/flare-floss#716]

**Comment by mr-tz** [comment]
From manual analysis of internal GP database:
- PE file format
  - DOS: This program cannot be run
	- exports: dll names and exports
	- meta: VS_VERSION_INFO
	- crt
	 - c and c++
- Delphi
- C++ regex library

**Comment by mr-tz** [comment]
With some parsing we can extract CRT strings (and code/function association) from `.lib`/`.obj` files.

**Comment by mr-tz** [comment]
for Delphi strings here's an existing database: https://github.com/dzzie/MAP/blob/master/delphi_filter.txt

**Comment by mr-tz** [comment]
Visual Studio contains source code (`.c`, `.cpp`, `.h`, `.asm`) as well as `.lib` and `.obj` files we can parse for run-time related strings.

**Comment by mr-tz** [comment]
@vee1e could we also use the OSS pipeline or something similar for extraction of strings from the runtime files?

**Comment by vee1e** [comment]
yeah, we'd just have to change the build steps and profile string updates like we did using vcpkg
